editor/src/main/com/mbrlabs/mundus/editor/core/project/ProjectManager.java

```diff
@@ -408,6 +408,9 @@ public void changeProject(ProjectContext context) {
             currentProject.dispose();
         }
 
+        //Clear out inspector
+        UI.INSTANCE.changeProject();
```

Comment:
This looks like it would work but I have a suggestion to try instead. Mundus uses events when possible to inform other classes of well, events. 

If you see line 429, `Mundus.INSTANCE.postEvent(new ProjectChangedEvent(context));` 

I think this can be done easier by undoing the changes to ProjectManager.java (since it already posts a event) and update the Inspector class to listen for the ProjectChangedEvent and call clearWidgets when the event is received. I think that should work the same and be a little cleaner. This way you don't need that other method in UI either. 

The Inspector class already implements some other events too. Here is a example of how to implement the ProjectChangedEvent event in GizmoManager

https://github.com/JamesTKhan/Mundus/blob/c14d9f3728516853e237d4f802c56fd724507553/editor/src/main/com/mbrlabs/mundus/editor/ui/gizmos/GizmoManager.kt#L12

```
class GizmoManager : ComponentAddedEvent.ComponentAddedListener,
        ComponentRemovedEvent.ComponentRemovedListener,
        SceneGraphChangedEvent.SceneGraphChangedListener,
        SceneChangedEvent.SceneChangedListener,
        ProjectChangedEvent.ProjectChangedListener {
 ```
        
 and
 ```
override fun onProjectChanged(event: ProjectChangedEvent) {
        clearGizmos()
}
```